src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -1107,6 +1107,15 @@
         az monitor log-analytics workspace table show --resource-group MyResourceGroup --workspace-name MyWorkspace -n MyTable
 """
 
+helps['monitor log-analytics workspace table create'] = """
+type: command
+short-summary: Create a Log Analytics workspace table..
+examples:
+  - name: Create a Log Analytics workspace table.
+    text: |
+        az monitor log-analytics workspace table create --resource-group MyResourceGroup --workspace-name MyWorkspace -n MyTable --retention-time 45 --columns MyColumn1=string
+"""
+
 helps['monitor log-analytics workspace table update'] = """
```

Comment:
Add examples with `--defaultDataCollectionRuleResourceId` as well for `update` command.